lesson07/assignment/assignment07/src/components/ContactPage.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div class="contact">
+
+    <body>
+      <main id="main-content" class="mb-5">
+        <h2 class="text-uppercase text-center mb-5 mt-4">Contact Us</h2>
+        <div class="mb-5 row justify-content-center">
+          <!-- RESPONSIVE GOOGLE MAP -->
+          <section class="col-12 col-md-6 col-lg-6 embed-responsive embed-responsive-4by3 m-2">
+            <iframe class="embed-responsive-item" src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2758.0094146662855!2d-118.15793938439825!3d46.26991718733962!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x54a2052ccce15555%3A0x3f76ee6658533a09!2sTen%20Ton%20Coffee!5e0!3m2!1sen!2sus!4v1611638602648!5m2!1sen!2sus" width="600" height="450" frameborder="0" style="border:0;" allowfullscreen="" aria-hidden="false" tabindex="0"></iframe>
+          </section>
+          <section class="contact-text col-12 col-md-5 col-lg-5 mt-0">
+            <address>
+              <p>216 Main St, Waitsburg, WA 99361</p>
+              <p><a class="contact-email contact-page-email-phone" href="mailto:tentoncoffeebar@gmail.com">tentoncoffeebar@gmail.com</a></p>
+              <p><a class="contact-page-email-phone" href="tel:555-555-5555">555-555-5555</a></p>
+            </address>
+          </section>
+        </div>
+      </main>
+    </body>
+  </div>
+</template>
+
+
+<script>
+export default {
+  name: 'Contact'
+}
+</script>
+
+<!-- Add "scoped" attribute to limit CSS to this component only -->
+<style scoped>
+
+</style>
```

Comment:
No need to include an empty style block. It can just be omitted if no scoped styles are needed.